tests/foreman/ui/test_gpgkey.py

```diff
@@ -1,9 +1,9 @@
 # -*- encoding: utf-8 -*-
 """Test class for GPG Key UI"""
 
+import random
```

Comment:
Ty @omaciel for sharing https://programmers.stackexchange.com/questions/187403/import-module-vs-from-module-import-function which has some interesting info about this.